octobot_trading/channels/orders.py

```diff
@@ -43,11 +43,12 @@ async def perform(self, orders, is_closed=False, is_from_bot=True):
                             order,
                             should_notify=False)
                     else:
-                        changed, is_updated = await self.channel.exchange_manager.exchange_personal_data.handle_order_update(
-                            symbol,
-                            order_id,
-                            order,
-                            should_notify=False)
+                        changed, is_updated = \
```

Comment:
Shouldn't we avoid having 2 return values? What do you think ?

Reply:
absolutely, I forgot to update this part, handle_order_update_from_raw is returning only one value